.semaphore/semaphore.yml

```diff
@@ -16,126 +16,152 @@ blocks:
             - script/lint_git
```

Comment:
At the top of this file I would add this warning so people are warned not to edit this generated file.

```yml
# DO NOT EDIT
# This is a generated file by the `TODO script name` script.
# See `.semaphore/semaphore.yml.erb` for the build matrix.
```